test/test_gemqrt.cc

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2017-2023, University of Tennessee. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#include "test.hh"
+#include "lapack.hh"
+#include "lapack/flops.hh"
+#include "print_matrix.hh"
+#include "error.hh"
+#include "lapacke_wrappers.hh"
+
+#include <vector>
+
+#if LAPACK_VERSION >= 30400  // >= 3.4.0
+
+//------------------------------------------------------------------------------
+// Simple overloaded wrappers around LAPACKE (assuming routines in LAPACKE).
+// These should go in test/lapacke_wrappers.hh.
```

Comment:
Can you move these 4 wrappers to lapacke_wrappers.hh?

Reply:
Done.